cirq/optimizers/decompositions_test.py

```diff
@@ -215,16 +215,18 @@ def test_single_qubit_matrix_to_native_gates_cases(intended_effect):
                           for _ in range(10)])
 def test_single_qubit_matrix_to_native_gates_fuzz_half_turns_always_one_gate(
         pre_turns, post_turns):
+    atol = 1e-6
+
     intended_effect = cirq.dot(
         cirq.unitary(cirq.Z**(2 * pre_turns)),
         cirq.unitary(cirq.X),
         cirq.unitary(cirq.Z**(2 * post_turns)))
 
     gates = cirq.single_qubit_matrix_to_phased_x_z(
-        intended_effect, atol=0.0001)
+        intended_effect, atol=atol)
```

Comment:
This tolerance refers to a per-gate effect, whereas the other tolerance refers to an aggregated effect. Add a safety factor of 10 to the aggregated one.

Reply:
Sounds good, done.